components/fxa-client/ios/FxAClient/FxAccountState.swift

```diff
@@ -86,6 +86,7 @@ extension FxAccountManager {
             }
         case .authenticatedWithProfile:
             switch event {
```

Comment:
You will need the `.fetchedProfile` transition otherwise we'll log an error: 
https://github.com/mozilla/application-services/blob/536a7f8c77430d9daeb68ad79568c048aca7b8bf/components/fxa-client/ios/FxAClient/FxAccountManager.swift#L308-L311

Reply:
added fetched profile below:
https://github.com/mozilla/application-services/pull/2950/commits/9b74e288a5ececb3d6e10d5f41a93bb6f073e2a6#diff-61b19ad6432d545017b667a4c94af2ceR90